package.json

```diff
@@ -154,13 +154,15 @@
     "collectCoverageFrom": [
       "src/**/*.{ts,tsx}",
       "!src/**/*Container.{ts,tsx}",
+      "!src/app/devices/saga.ts",
```

Comment:
I'd rather not skip adding tests for this.

Reply:
This is the root saga, so there isn't anything to test here:

export default function* rootSaga() {
    yield all([
        ...deviceListSagas,
        ...deviceContentSagas,
        ...notificationsSagas,
    ]);
}

Reply:
Added it back, since we do want to test the root saga. Going to really test it in a different PR afterwards. @montgomp